inc/command/class-command.php

```diff
@@ -41,12 +41,12 @@ protected function configure() {
                                 if you are running Local Chassis.
 
 To run Codeception integration tests:
-    codecept [--chassis] -m <module> -b <browser> [--] [options]
+    codecept [--chassis] -p <path> -b <browser> [--] [options]
                                 use `--` to separate arguments you want to
                                 pass to Codeception. Use the --chassis option
-                                if you are running Local Chassis. Use -m module
-                                to test an Altis module. Use --browser/-b to run
-                                a headless browser container for acceptance tests,
+                                if you are running Local Chassis. Use -p path
+                                to specify custom tests folder. Use --browser/-b
+								to run a headless browser container for acceptance tests,
                                 choose 'chrome', 'firefix', or 'edge' as needed.
```

Comment:
```suggestion
                                choose 'chrome', or 'firefox' as needed.
```
no edge support yet right?